### PR TITLE
Mixpanel group call updates

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Group key updates

### CODE REVIEW

**Review by shrouti1507 — APPROVED** [review]

File: docs/destinations/streaming-destinations/mixpanel.mdx [hunk 1]
```diff
@@ -68,7 +68,7 @@ The following settings are applicable if you are using this destination via the
 - **Cross Subdomain Cookie**: If enabled, this option lets the Mixpanel cookie to persist between different pages of your application.
 - **Persistence Type**: This option lets you choose the persistence type for your Mixpanel cookies.
 - **Secure Cookie**: Enabling this option will mark the Mixpanel cookie as secure, i.e., it will only transmit over HTTPS.
-- **Group Key Settings**: RudderStack sends the `group` calls to Mixpanel only if one or more group keys are specified here. For more information, refer to the <Link to="#group"> Group</Link> section below.
+- **Group Key Settings**: RudderStack sends the `group` calls to Mixpanel only if one or more group keys are specified here. These group keys act as group identifiers in Mixpanel. For more information, refer to the <Link to="#group-key">Group Key</Link> section below.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mixpanel.mdx:71]:
```suggestion
- **Group Key Settings**: RudderStack sends the `group` calls to Mixpanel only if one or more group keys are specified here. These group keys act as the group identifiers in Mixpanel. For more information, refer to the <Link to="#group-key">Group Key</Link> section below.
```

File: docs/destinations/streaming-destinations/mixpanel.mdx [hunk 2]
```diff
@@ -285,14 +280,22 @@ rudderanalytics.group(
 );
 ```
 
-RudderStack sends the `group` calls to Mixpanel only if one or more group keys are specified during the destination configuration in RudderStack, as shown:
+### Group Key
+
+You can create group keys in [Mixpanel project settings](https://help.mixpanel.com/hc/en-us/articles/360025333632-Group-Analytics#implementation), which act as group identifiers in Mixpanel. To send a `group` call to Mixpanel successfully, you must specify **at least** one group key (must be present in Mixpanel) in the **Group Key** RudderStack dashboard setting, as shown:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mixpanel.mdx:285]:
```suggestion
You can create group keys in the [Mixpanel project settings](https://help.mixpanel.com/hc/en-us/articles/360025333632-Group-Analytics#implementation), which act as the group identifiers in Mixpanel. To successfully send a `group` call to Mixpanel, you must specify **at least** one group key in the **Group Key** RudderStack dashboard setting, as shown:
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/mixpanel.mdx:285]: removed `(must be present in Mixpanel)` as it is obvious from the previous sentence.

**Review by ameypv-rudder — APPROVED** [review]
Some minor suggestions but LGTM overall.

**Pull request merged**